cli/slsa-verifier/main.go

```diff
@@ -41,21 +44,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	var ptag, pversiontag *string
+	var pbuilderID, ptag, pversiontag *string
 
 	if isFlagPassed("tag") {
 		ptag = &tag
 	}
 	if isFlagPassed("versioned-tag") {
```

Comment:
> comment from @ianlewis: 

aside: I'm curious. Remind me again why these flags are copied to local variables? Also, why we need a special isFlagPassed function?

> reply from @laurentsimon 
I don't think I found a way to have an option be a pointer to a string, and for the pointer to be nil if the user does not provide the option. If that's possible, I can update the code.

Reply:
Yeah, I believe `StringVar` is as close as you can get, and the best that can be done is use a default empty string. but problem is that empty string doesn't necessarily mean no value.

In this case we can document that? empty tags passed in mean no tag verification.

Reply:
done. Added a comment.